api/src/tarball.rs

```diff
@@ -158,6 +158,12 @@ pub async fn process_tarball(
       });
     }
 
+    if path.ends_with(".cjs") | path.ends_with(".cts") {
+      return Err(PublishError::BannedCommonJsExtension {
+        specifier: path.to_string(),
+      });
+    }
```

Comment:
I don't think we should do this here - users may have `.cjs` files in the tarball they are publishing, even if that `.cjs` file is not ever imported (for example because it is part of the projects' tests).

Can you remove this check, and only keep the other check?

Reply:
ah, yes. absolutely makes sense, thanks for the hint!